src/meta/user/authorizer/index.ts

```diff
@@ -7,6 +7,16 @@ import { User } from 'meta/user/user'
 import { Collaborator, CollaboratorEditPropertyType } from 'meta/user/userRole'
```

Comment:
Only minor code quality added:
- new function in separate file: `canViewReview`
- use `AuthProps` prop type where feasible (1:1)
- use `Users.getUserRoles` where more than one role is checked